packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -146,6 +148,11 @@ export const AuthContextProvider = ({
     setLoginState({ trigger: AuthTriggers.LegacyLogout });
   }
 
+  const deleteAccount = async () => {
+    await dispatchDeleteAccount();
+    updateLocalBootThemeData?.();
```

Comment:
I think it is too specific. We can probably just pass the whole update settings function so we won't have to introduce another function when we need to implement something similar in the future.

Reply:
@sshanzel actually after a discussion with Chris, we don't need to keep any boot data when an account is deleted, so I'll move on removing the whole thing rather then updating.